configure.ac

```diff
@@ -29,7 +29,8 @@ AC_ARG_WITH([libfabric],
             AC_HELP_STRING([--with-libfabric], [Use non-default libfabric location - default NO]),
             [AS_IF([test -d $withval/lib64], [fab_libdir="lib64"], [fab_libdir="lib"])
              CPPFLAGS="-I $withval/include $CPPFLAGS"
-             LDFLAGS="-L$withval/$fab_libdir $LDFLAGS"],
+             LDFLAGS="-L$withval/$fab_libdir $LDFLAGS"
+	     CFLAGS=$withval/$fab_libdir/libfabric.a],
```

Comment:
> I'm not sure I understand the CFLAGS value here. Did you just want libfabric, not libfabric.a?

I did want libfabric.a so that buddy_test.c is statically linked to the required object files in this archive, but I'm not sure about these configury changes. I was following the instructions listed here (https://github.com/ofi-cray/libfabric-cray/wiki/Building-and-profiling-with-gperftools) in order to compile buddy_test.c with gperftools.

Perhaps we should leave these configury changes out and remove buddy_test.c from Makefile.am?  This way, the user would have to compile buddy_test.c manually.